docs/release/checklist.txt

```diff
@@ -0,0 +1,32 @@
+Release checklist
+-----------------
+
+- Clone the release branch - do `git branch -u` to the upstream branch, so an update and rebase will get you to latest.
+
+- Build an RPM in the COPR -test repo (e.g. pbench-0.72-test).
+
+-  Create VMs of various stripes for testing and prep them appropriately (e.g. ssh keys need to be installed for ansible to work and repos need to be installed (e.g CRB repos on RHEL-8 and RHEL-9 and EPEL repos).
+
+- Run the playbooks to do the prep, repo install, pbench-agent/pbench-systat install, the config of the agent and the firewall rules.
+
+- Run end-to-end tests of linpack, uperf, fio: register tools, run the benchmark, clean up the tools, push the results. There is also a smoke test that runs a quick pbench-user-benchmark.
+
+- Lather, rinse, repeat (steps 1-5) until there are no errors.
+
+- Write up release notes - submit for review and revise.
+
+- Make sure there are no pending updates to the release branch.
+
+- Add a tag and cut the release on GitHub, adding the release notes.
+
+- Redo steps 2-5 in the updated branch.
+
+- Rebuild RPM in the COPR repo for the release (e.g. pbench-0.72) and redo steps 4-5 (the smoke test in particular).
+
+- Build tagged containers.
```

Comment:
This harks back to your comments in scrum: it might not be a bad idea to add actual command syntax to some of these bullet points while they're fresh in your mind as an aid to future developers who tread this path.